.github/assets/file_loader_icon.svg

```diff
@@ -1,36 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg width="200px" height="200px" viewBox="0 0 200 200" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
@bebraw Sorry, are you using it?
I changed just to show in an example. I looked for references to him before, I didn't find so
I didn't think it could be a problem.